src/System.Web.HttpUtility/src/System/Web/Util/HttpEncoder.cs

```diff
@@ -283,45 +283,14 @@ internal static byte[] UrlDecode(byte[] bytes, int offset, int count)
                 }
                 else if (b == '%' && i < count - 2)
                 {
-                    if (bytes[pos + 1] == 'u' && i < count - 5)
-                    {
-                        int h1 = HttpEncoderUtility.HexToInt((char)bytes[pos + 2]);
```

Comment:
I do not. I ported the code from the internal branch not from reference source so it could have been changed recently.